ui/src/components/auth/LoginContainer.js

```diff
@@ -30,47 +28,33 @@ const LoginContainer = (props) => {
 
   const postLogin = async (email, password) => {
     try {
-      const { data } = await axios.post("/api/auth/login", {
+      const { data } = await axios.post('/api/auth/login', {
         emailAddress: email,
         password: password,
       });
       dispatch(signin(data));
-      setInvalidLogin("");
+      setInvalidLogin('');
       window.location.reload();
     } catch (err) {
       if (err.response.status === 500) {
```

Comment:
Might wanna add an extra condition. Realized that if the email is bad (non existent email) it returns a error code 401. Causing this line to not trigger and no error message to be set, which can be confusing for the user.